abis/DepositManager.json

```diff
@@ -200,6 +200,12 @@
     "name": "DepositRetrieved",
     "type": "event"
   },
+  {
+    "anonymous": false,
+    "inputs": [],
+    "name": "EIP712DomainChanged",
+    "type": "event"
+  },
```

Comment:
Lmao this is my bad, guess I never rebuilt subgraph with new abis

Reply:
Apparently behavior is just that it doesn't pick those events up, not error.